superset-frontend/src/explore/controlUtils.js

```diff
@@ -141,8 +139,9 @@ export function getControlStateFromControlConfig(controlConfig, state, value) {
   // If a choice control went from multi=false to true, wrap value in array
   const controlValue =
     controlConfig.multi && value && !Array.isArray(value) ? [value] : value;
+
   controlState.value =
-    typeof controlValue === 'undefined' ? controlState.default : controlValue;
+    controlValue === undefined ? controlState.default : controlValue;
```

Comment:
no need to change.

Reply:
thought this change will make the code short enough to fit in one line... obviously not...